src/gtk/toga_gtk/font.py

```diff
@@ -32,8 +33,32 @@ def __init__(self, interface):
         try:
             font = _FONT_CACHE[self.interface]
         except KeyError:
-            font = Pango.FontDescription.from_string('{font.family} {font.size}'.format(font=self.interface))
-            _FONT_CACHE[font] = font
+
+            # Initialize font with properties 'None NORMAL NORMAL NORMAL 0'
+            font = Pango.FontDescription()
+
+            # Set font family
+            # TODO: Check whether font is installed in system
+            font.set_family(self.interface.family)
+
+            # Set font size
+            font.set_size(self.interface.size)
+
+            # Set font style
+            if self.interface.style is ITALIC:
+                font.set_style(Pango.Style.ITALIC)
+            elif self.interface.style is OBLIQUE:
+                font.set_style(Pango.Style.OBLIQUE)
+
+            # Set font variant
+            if self.interface.variant is SMALL_CAPS:
+                font.set_variant(Pango.Variant.SMALL_CAPS)
+
+            # Set font weight
+            if self.interface.weight is BOLD:
+                font.set_weight(Pango.Weight.BOLD)
+
+            _FONT_CACHE[self.interface] = font
```

Comment:
What's the failure mode here? On other platforms, requesting a font that doesn't exist still results in returning a font object - it just falls back to the system font, logging the problem. Does this API result in a usable font object if you ask for a non-existent font?

Reply:
In a way yes, but doesn't warn about the problem? afaik Pango doesn't do error handling, however Gtk's Font chooser interface might help... But I really don't know.

Reply:
@freakboy3742 I made changes for explicitly making font default to `system`, on a side note I have a single font installed in my system(Noto) and all the families work correctly including `Cursive` and `Fantasy`.